src/services/Models.service.tsx

```diff
@@ -79,7 +79,10 @@ export const normaliseMaterialsConfig = (materials: Record<string, Material>, bl
     if (mat.map) {
       mat.map.minFilter = LinearFilter;
       mat.depthWrite = true;
-      mat.toneMapped = false;
+    }
+
+    if (mat.name.toLowerCase().includes('hair')) {
+      mat.roughness = 0.9;
```

Comment:
I'm wondering how come we have to modify roughness for hair explicitly?

Reply:
During mobtesting we noticed that the hair looked very plasticy with the new rim lights. We decided to go for an easy adjust here in order to avoid updating the assets. I think the defined value is something like .7 - .8 so it is only a minor tweak with decreasing the left light intensity.